src/NUnitConsole/nunit3-console/TestEventHandler.cs

```diff
@@ -42,6 +42,9 @@ public class TestEventHandler : MarshalByRefObject, ITestEventListener
         private readonly bool _displayAfterTest;
         private readonly bool _displayBeforeOutput;
 
+        private string _currentTestName;
```

Comment:
Ok, will change the name.

_currentTestName was chosen because it is often initialised with the same value as _currentLabel.